src/streamlink/plugins/gardenersworld.py

```diff
@@ -4,26 +4,27 @@
 
 from streamlink import NoPluginError
 from streamlink.plugin import Plugin
+from streamlink.plugin.api import http
```

Comment:
`http` can be removed, it got added with a rebase.

Reply:
True that, forgot to clean up after I rebased. I saw it (using PyCharm to do the rebase), and i was going to tidy it up but I forgot :)